atest/robot/running/continue_on_failure_tag.robot

```diff
@@ -0,0 +1,70 @@
+*** Settings ***
+Suite Setup       Run Tests    ${EMPTY}    running/continue_on_failure_tag.robot
+Resource          atest_resource.robot
+
+*** Test Cases ***
+Continue in test with tag
+    ${tc}=    Check Test Case    ${TESTNAME}
```

Comment:
`${tc}=    ` part is unnecessary when no further validation is done for these tests.